examples/common/c/dictionary.c

```diff
@@ -1,8 +1,8 @@
 #include <ctype.h>
 #include <string.h>
 
+#include "../../../src/c/util/rmemory.h"
```

Comment:
If you're removing the rasta-specific memory management functions now, you should do it consistently.